dll/win32/comctl32/listview.c

```diff
@@ -12001,7 +12001,30 @@ LISTVIEW_WindowProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
       }
       return DefWindowProcW(hwnd, uMsg, wParam, lParam);
 
+#ifdef __REACTOS__
+  case WM_SETTINGCHANGE: /* Same as WM_WININICHANGE */
+    if (wParam == SPI_SETICONTITLELOGFONT)
+    {
+      BOOL bWasSameFont = (infoPtr->hDefaultFont == infoPtr->hFont);
```

Comment:
Or this:
```suggestion
      BOOL bWasSameFont = ((infoPtr->hDefaultFont == infoPtr->hFont) || !infoPtr->hFont);
```
("bWasSameFont" --> "bIsDefaultFont" ?)

Reply:
`bDefaultOrNullFont`?